media/css/firefox/accounts.scss

```diff
@@ -0,0 +1,326 @@
+// Common elements
+
+main {
+    background: #003eaa;
+    background: url('/media/img/firefox/accounts/tail.png') top -350px center no-repeat,
+                linear-gradient(to bottom, #003eaa 0%, #00b0ff 100%);
+    color: #fff;
+}
+
+/* -------------------------------------------------------------------------- */
+// Top header logos
+
+.top-header {
+    text-align: center;
+
+    h1 {
+        @include font-size-level2;
+        margin: 40px 0;
+    }
+
+    .header-logos {
+        margin-bottom: 20px;
+
+        .mozilla {
+            display: none;
+        }
+    }
+
+    @media #{$mq-desktop} {
+
+        h1 {
+            margin: 40px 0 80px;
+        }
+
+        .header-logos {
+            @include clearfix;
+
+            .firefox {
+                float: left;
+            }
+
+            .mozilla {
+                display: block;
+                float: right;
+            }
+        }
+    }
+}
+
+html[dir="rtl"] .top-header {
+    @media #{$mq-desktop} {
+        .header-logos {
+
+            .firefox {
+                float: right;
+            }
+
+            .mozilla {
+                float: left;
+            }
+        }
+    }
+}
+
+/* -------------------------------------------------------------------------- */
+// Main page content container
+
+.main-content {
+    @include clearfix;
+    margin-bottom: 100px;
+
+    @media #{$mq-desktop} {
+        min-height: 400px;
+    }
+}
+
+/* -------------------------------------------------------------------------- */
+// Main page copy
+
+.main-copy {
+    @include background-size(280px, 122px);
+    background-image: url('/media/img/firefox/accounts/sync-devices.png');
+    background-position: top center;
+    background-repeat: no-repeat;
+    margin: 0 auto;
+    max-width: 400px;
+    padding-top: 140px;
+    text-align: center;
+
+    .tagline {
+        @include font-size-level4;
+    }
+
+    ol {
+        list-style: decimal;
+        list-style-position: inside;
+
+        li {
+            margin-bottom: 10px;
+        }
+    }
+
+    @media #{$mq-desktop} {
+        @include background-size(350px, 153px);
+        background-position: top left;
+        float: left;
+        max-width: 513px;
+        padding-top: 170px;
+        text-align: left;
+        width: calc(100% - 470px);
+    }
+
+    @media #{$mq-desktop-wide} {
+        @include background-size(513px, 224px);
+        padding-top: 240px;
+        width: calc(100% - 500px);
+    }
+}
+
+html[dir="rtl"] .main-copy {
+    @media #{$mq-desktop} {
+        background-position: top right;
+        float: right;
+        text-align: right;
+    }
+}
+
+/* -------------------------------------------------------------------------- */
+// Firefox Accounts for container
+
+.fxa-cta {
+    background: #fff;
+    background: #fff;
+    border-radius: 6px;
+    margin: 0 auto;
+    max-width: 400px;
+    text-align: center;
+    padding-top: 40px;
```

Comment:
Might need some `padding-bottom` on Fx at < 1000px wide, as the button is flush against the bottom of the white box.

![screen shot 2018-02-20 at 10 44 23 am](https://user-images.githubusercontent.com/317532/36438738-6474347c-1630-11e8-80e3-a7dfd8c39386.png)

(Going between breakpoints results in a not great UX, but I believe this has been the case for quite some time due to how the iframe resizes itself. Probably still an edge case for now.)

Reply:
I think setting a `min-height` on the iframe itself here might help, and also fix the issue when switching between breakpoints.